_unittest/test_26_emit.py

```diff
@@ -650,11 +643,11 @@ def test_N_to_1_feature(self):
         assert instance.get_largest_problem_type(econsts.result_type().emi) == "Out-of-Channel: Tx Fundamental"
         domain2 = self.aedtapp.results.interaction_domain()
         rx_frequencies = rev.get_active_frequencies(radiosRX[0], bandsRX[0], econsts.tx_rx_mode().rx, "Hz")
-        domain2.set_receiver(radiosRX[0], bandsRX[0], rx_frequencies[0])
+        domain2.set_receiver(radiosRX[0], bandsRX[0], rx_frequencies[0], "MHz")
```

Comment:
Are we retrieving rx_frequencies in Hz but setting them in MHz?  Original change just used Hz.

Reply:
@apollo-crum good catch